e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubPublisherDeployment.cs

```diff
@@ -0,0 +1,205 @@
+            if (isLinux) {
+                if (isStandalone) {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        Hostname = "publisher_standalone",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa",
+                        "--pf=/mount/published_nodes.json"
+                    },
+                        HostConfig = new {
+                            Binds = new[] {
+                            "/mount:/mount"
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
+                else {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        Hostname = "publisher",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa"
+                    },
+                        HostConfig = new {
+                            Binds = new[] {
+                            "/mount:/mount"
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
+            }
+            else {
+                if (isStandalone) {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        User = "ContainerAdministrator",
+                        Hostname = "publisher_standalone",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa",
+                        "--pf=C:\\\\mount\\\\published_nodes.json"
+                    },
+                        HostConfig = new {
+                            Mounts = new[] {
+                            new {
+                                Type = "bind",
+                                Source = "C:\\\\ProgramData\\\\iotedge",
+                                Target = "C:\\\\mount"
+                                }
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
+                else {
+                    createOptions = JsonConvert.SerializeObject(new {
+                        User = "ContainerAdministrator",
+                        Hostname = "publisher",
+                        Cmd = new[] {
+                        "PkiRootPath=/mount/pki",
+                        "--aa"
+                    },
+                        HostConfig = new {
+                            Mounts = new[] {
+                            new {
+                                Type = "bind",
+                                Source = "C:\\\\ProgramData\\\\iotedge",
+                                Target = "C:\\\\mount"
+                                }
+                            }
+                        }
+                    }).Replace("\"", "\\\"");
+                }
+            }
```

Comment:
Can be removed, we don't plan to use windows as OS for edge device for testing

Reply:
fixed